articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+---
+title: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+description: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Site Recovery
+toc_sub3:
+toc_sub4:
+toc_title: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+toc_fullpath: Operators/How To/azs-how-setup-recovery-portal.md
+toc_mdlink: azs-how-setup-recovery-portal.md
+---
+# Setting up disaster recovery for Azure Stack VMs to Azure via the portal
+
+## Overview
+
+The purpose of this article is to help with the setting up of the **Azure Stack** disaster recovery to **Public Azure** using the **Azure Site Recovery Service**.
+
+Site Recovery once set up helps contribute to your business continuity. The service ensures that VM workloads you have protected will remain available when expected and unexpected outages occur, by failing them over to **Public Azure**. You are able to then import the VM's back to **Azure Stack** once platform issues have been resolved.
+
+* Site Recovery manages replication of VMs to Public Azure.
+
+* When an outage occurs in your primary site, you are use Site Recovery to fail over to Public Azure.
```

Comment:
typo: you are use
Not sure you need this paragraph though as it just repeats information that you already mentioned above

Reply:
done